docs/src/network.md

```diff
@@ -631,7 +631,7 @@ async def handle_route(route: Route) -> None:
         # Override response body.
         body=body,
         # Force content type to be html.
-        headers={**response.headers, "content-type": "text/html"},
+        headers={**response.all_headers(), "content-type": "text/html"},
```

Comment:
Needs await.

Reply:
Reverted since the response still has `headers`.